### DISCUSSION TITLE
School uses macbooks.

### DISCUSSION THREAD

**Discussion by Endy3838** [3kh0/ext-remover#1054] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
my school currently use macbook air's m1. and im currently asking for any tips to disable their spyware

**Comment by ohiobombs** [comment]
dont worry about it and enjoy your school year of richness

**Comment by nathblade16** [comment, marked as answer]
try to install asahi Linux if you really want to
